lighthouse-core/test/lib/network-recorder-test.js

```diff
@@ -17,6 +18,33 @@ describe('network recorder', function() {
     assert.equal(records.length, 76);
   });
 
+  it('handles redirects properly', () => {
```

Comment:
it feels like we should be adding a lot more tests here, though I realize our tests for working network request creation were just as implicit before :) WDYT? Can we cheat and bring over devtools tests or are they all in the form of crazy layout tests?

Reply:
> Can we cheat and bring over devtools tests

there are devtools tests!? 😉

there's really nothing portable, I have a proposal for "supplemental" testing based on the lantern-data directory and using jest we can discuss tomorrow at eng sync, but our extensive use of network records throughout the rest of the codebase was really quite good coverage

Reply:
> but our extensive use of network records throughout the rest of the codebase was really quite good coverage

yeah, they are pretty fundamental. I'm ok with that + incrementally adding as we hit issues/update